src/Microsoft.Identity.Client/Cache/CacheFallbackOperations.cs

```diff
@@ -32,16 +32,17 @@
 using Microsoft.Identity.Client.Cache.Items;
 using Microsoft.Identity.Client.Core;
 using Microsoft.Identity.Client.Exceptions;
+using Microsoft.Identity.Client.Instance;
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Utils;
 
 namespace Microsoft.Identity.Client.Cache
 {
     internal static class CacheFallbackOperations
     {
-        internal /* internal for testing only */ const string DifferentEnvError = 
+        internal /* internal for testing only */ const string DifferentEnvError =
```

Comment:
are these error messages used?  they're not referenced in this file.  also, shouldn't these be on MsalErrorMessage?

Reply:
No, these are log messages. Since it was decided to try / catch all on CacheFallback logic, the only way to unit test is to assert that some log messages are sent through.